client/src/components/Gallery.jsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+
+class Gallery extends React.Component {
+    render() {
+        // if (this.props.data.length > 0) {
+        //     var image1 = this.props.data[0].Picture[0]; //works
+        //     var price1 = this.props.data[0].Price;
+        // };
+        return (
+            <div>
+
+                <h1>More homes you may like</h1>
+                <div>footer info</div>
+                <ul>
+                    <li><a href='#'>Become a Host</a></li>
+                    <li><a href='#'>Help</a></li>
+                </ul>
+                <div
+                    style={{
+                        width: '200px',
+                        height: '80px',
+                        backgroundColor: 'yellow',
+                        fontSize: '24px',
+                        textAlign: 'center'
+                    }}
```

Comment:
Move this style as class to css file, avoid using CSS inline style

Reply:
got it, it is for testing, i will clear all of it in later version